.github/workflows/linux.yml

```diff
@@ -0,0 +1,102 @@
+name: linux
+
+on: 
+  push:
+    branches: [dev_wd]
```

Comment:
does this have to be updated to include other private branches, develop and master?

Reply:
The on push just makes sure tests are run when a specific branch is, which was handy to develop them. For us the 'on pullrequest' is more important. I removed on push in 870d381